katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -160,18 +170,22 @@ exit 0
 /var/cache/katello-agent/
 %attr(750, root, root) %{_sbindir}/katello-package-upload
 %attr(750, root, root) %{_sbindir}/katello-enabled-repos-upload
-%{_prefix}/lib/yum-plugins
```

Comment:
Would it make sense to keep this but add an ```%exclude %{_prefix}/lib/yum-plugins/tracer_upload.py``` and would that even work?

Reply:
I honestly don't know without further testing.

Reply:
I tend to be more explicit with these things.  For example if i added a new plugin in a new subpackage, i'd have to remember to exclude it there. But I can see advantages to both